tests/modules/services/window-managers/sway/sway-followmouse-legacy.nix

```diff
@@ -6,7 +6,7 @@ with lib;
   config = {
     wayland.windowManager.sway = {
       enable = true;
-
+      package = pkgs.writeScriptBin "sway" "" // { outPath = "@sway"; };
```

Comment:
These should almost certainly be "@sway@". Could we change this at the same time?